src/frontend/src/components/NERAutocomplete.tsx

```diff
@@ -81,7 +99,7 @@ const NERAutocomplete: React.FC<NERAutocompleteProps> = ({
       id={id}
       onChange={onChange}
       options={options}
-      sx={autocompleteStyle}
+      sx={chooseStyle}
```

Comment:
this component was redesigned already in develop, I would merge the changes from develop into this branch to get the updated code